exercises/sheet4/marta_bustins_marta_gonzalez_esteve_tarrago/4.tex

```diff
@@ -0,0 +1,186 @@
+\end{obs}
+
+%\textcolor{red}{crec que falta teorema al que es refereix la prova} \textcolor{blue}{A veure, aquí la intenció era posar el teorema TEOREMA, en plan abans de posar-lo primer fixem nocions i fem alguna observació,i després doncs TEOREMA: bla bla bla DEMOSTRACIÓ: bla bla bla perquè crec que quedaria una mica més endraçat... Però si teniu altres opcions endavant!}
+
+
+
+\textit{Proof}. Let $\gamma$ and $\rho$ be two sections of $P$, and let $\lambda\in\mathbb{R}$. Then $\lambda\gamma+\left(1-\lambda\right)\rho$ is continuous since it is a linear combination of two continuous functions, and 
+\begin{align*}
+    \pi\left(\lambda\gamma+\left(1-\lambda\right)\rho\right)\left(x\right)&=\pi\left(\lambda\gamma\left(x\right)+\left(1-\lambda\right)\rho\left(x\right)\right)\\
+    &=\lambda\pi\left(\gamma\left(x\right)\right)+\left(1-\lambda\right)\pi\left(\rho\left(x\right)\right)\quad\qquad\mathrm{Since\,}\pi\mathrm{\,is\,linear}\\
+    &=\lambda x+\left(1-\lambda\right)x\qquad\qquad\qquad\qquad\quad\;\mathrm{Since\,}\gamma\mathrm{\,and\,}\rho\mathrm{\,are\,sections}\\
+    &=x\\
+\end{align*}
+so any convex combination of sections is a section (seen just by 2 sections, but repeating this argument by induction can be easily proved). In particular, this shows that $\Sigma\left(P,Q\right)$ is convex, since for any $x$, $y\in\Sigma\left(P,Q\right)$, $x=\frac{1}{\mathrm{volQ}}\int_Q\gamma\left(s\right)ds$, $y=\frac{1}{\mathrm{volQ}}\int_Q\rho\left(s\right)ds$, $\gamma$, $\rho$ two sections and $\lambda\in\left[0,1\right]$ we get that
+    $$\lambda\frac{1}{\mathrm{volQ}}\int_Q\gamma\left(s\right)ds+\left(1-\lambda\right)\frac{1}{\mathrm{volQ}}\int_Q\rho\left(s\right)ds=\frac{1}{\mathrm{volQ}}\left(\int_Q\left(\lambda\gamma\left(s\right)\left(1-\lambda\right)\rho\left(s\right)\right)ds\right)\in\Sigma\left(P,Q\right)$$
+since convex combinations of sections is a section, as we have seen. Moreover, $\Sigma\left(P,Q\right)\subseteq\pi^{-1}\left(r_0\right)$ since, for any $x\in\Sigma\left(P,Q\right)$, $x=\frac{1}{\mathrm{volQ}}\int_Q\gamma\left(s\right)ds$, $$\pi\left(x\right)=\pi\left(\frac{1}{\mathrm{volQ}}\int_Q\gamma\left(y\right)dy\right)=\frac{1}{volQ}\int_Q\pi\left(\gamma\left(y\right)\right)dy=\frac{1}{volQ}\int_Qydy=r_0.$$ On the other hand, $\pi\colon\mathbb{R}^p\to\mathbb{R}^q$ as afine function can be written as $\pi=\pi_l+r_0$, where $\pi_l\colon\mathbb{R}^p\to \mathbb{R}^q$ is a linar function. Hence, in order to find $\mathrm{dim}\pi^{-1}\left(r_0\right)$ it suffices to find $\mathrm{dim}\pi^{-1}_l\left(0\right)=\mathrm{dim\,ker}\pi_l$. By the first isomorphism theorem we have $$\mathrm{dim\,ker}\pi_l=\mathrm{dim\,P}-\mathrm{dim\,Q}.$$ Thus, $\Sigma\left(P,Q\right)$ has dimension $\mathrm{dim\,P}-\mathrm{dim\,Q}$ at most.
+
+Since tight sections correspond to some subsets of $\mathcal{L}\left(P\right)$, there will be a finite number of tight sections.
```

Comment:
This is only true for affine sections. If the section isn't affine, it isn't determined just by its domain or range